frontend/public/components/RBAC/bindings.jsx

```diff
@@ -113,9 +119,12 @@ const Row = ({obj: binding}) => <ResourceRow obj={binding}>
   <OverflowYFade className="col-md-2 hidden-sm hidden-xs">
     {binding.subject.name}
   </OverflowYFade>
-  <OverflowYFade className="col-md-2 col-sm-4 col-xs-6 co-break-word">
+  <OverflowYFade className="col-md-2 col-sm-3 col-xs-5 co-break-word">
     {binding.metadata.namespace ? <ResourceLink kind="Namespace" name={binding.metadata.namespace} /> : 'all'}
   </OverflowYFade>
+  <div className="col-sm-1 col-xs-2">
```

Comment:
> Moving the actions away from the resource they're for gives me pause. With the existing design, it's very easy to make the association between the resource and the actions that belong to it because they are side-by-side.

Here's a specific example where it might be a problem. In a table like this, it could look like the actions are for the service instance instead of the binding if we pull the actions menu to the right.

![service bindings okd 2018-08-20 14-31-31](https://user-images.githubusercontent.com/1167259/44359415-d53dc180-a485-11e8-9183-a1441d18d91f.png)